package.json

```diff
@@ -61,8 +56,9 @@
     "lint-js": "./node_modules/.bin/eslint \"media/js/**/*.js\" \"tests/unit/spec/**/*.js\" webpack.config.js webpack.static.config.js",
     "lint-css": "./node_modules/.bin/stylelint \"media/css/**/*.{css,scss}\"",
     "lint": "npm run lint-js && npm run lint-css && npm run prettier-check",
-    "test": "npm run glean && npm run lint && npm run karma",
-    "karma": "./node_modules/.bin/karma start ./tests/unit/karma.conf.js",
+    "test-build": "webpack --config webpack.test.config.js --mode development",
+    "pretest": "npm run glean && npm run lint && npm run test-build",
+    "test": "./node_modules/jasmine-browser-runner/bin/jasmine-browser-runner runSpecs --config=tests/unit/jasmine-browser.json",
```

Comment:
It would be nice to have a command that only runs the tests, without the glean and linting pre-step? This is something I often use for dev when writing tests as it's a bit faster to run. Something like `npm run jasmine` might be nice? And then `npm test` would be the full set of checks including linting and formatting etc?

```
"test-build": "webpack --config webpack.test.config.js --mode development",
"test": "npm run glean && npm run lint && npm run test-build && npm run jasmine",
"jasmine": "npm run test-build && ./node_modules/jasmine-browser-runner/bin/jasmine-browser-runner runSpecs --config=tests/unit/jasmine-browser.json",
```